install_deps.sh

```diff
@@ -53,6 +53,7 @@ apt-get ${APT_GET_FLAGS} install \
     libzstd-dev
 
 pip3 install --upgrade wheel && pip3 install --upgrade trio
+pip3 install eliot eliot-tree
```

Comment:
AFAIK, if you are adding dependencies, you need to generate a new version of the concord-bft image. Can you please confirm with Dima?

Reply:
Sure